### PR TITLE
UI tweaks & bug fix

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 1]
```diff
@@ -12,22 +12,12 @@ <h2 class="primary-display">{{title}}</h2>
       <span class="spinner spinner-sm hidden-sm-up"></span>
     </div>
   </div>
+  <a class="btn btn-link btn-sm" [routerLink]="['']">< Back to main search </a>
   <div class="result-list">
     <div class="db-card">
       <div class="db-card-inner">
         <section *ngIf="items.length > 0">
           <!-- Top summary boxes -- only show if have results -->
-          <div class="domain-summary">
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:20]: remove redundant

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 2]
```diff
@@ -196,8 +183,8 @@ <h5 id="domain-name" class="primary-display" *ngIf="searchText.value.length > 0"
             </div>
             <div (click)="expandRow(standardConcepts.concat(items),r)" class="secondary-display info-text aka"
               [ngClass]="{'no-b-border': r.expanded}" *ngIf="r.conceptSynonyms;else noConceptSynonyms">
-              <div (click)="$event.stopPropagation();toggleSynonyms(r.conceptId)" class="aka-text"
-                [ngClass]="{'no-b-border': r.expanded}"><span>Also Known As</span>
+              <div *ngIf="synonymString[r.conceptId]" (click)="$event.stopPropagation();toggleSynonyms(r.conceptId)" class="aka-text"
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:181]: show aka text only if there are synonyms

File: public-ui/src/app/views/survey-view/survey-view.component.html [hunk 3]
```diff
@@ -13,7 +13,7 @@ <h1> {{survey.name}} </h1>
       <clr-icon shape="search" size="25" class="is-info search-icon"></clr-icon>
       <input type="text" id="search-db" placeholder="Keyword Search" [formControl]="searchText" autofocus>
     </div>
-
+    <a class="btn btn-link btn-sm" [routerLink]="['']">< Back to main search </a>
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/survey-view/survey-view.component.html:16]: back to search link

File: public-ui/src/app/styles/page.css [hunk 4]
```diff
@@ -109,7 +109,7 @@
 }
 
 .search-bar-container {
-  margin-bottom: calc( 18px *4);
+  margin-bottom: calc( 18px * 2);
```
Comment (mhigbyflowers) [inline public-ui/src/app/styles/page.css:112]: search bar margin now matches mockup

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 5]
```diff
@@ -114,7 +114,7 @@ export class ChartComponent implements OnChanges {
         pie: {
           borderColor: null,
           slicedOffset: 4,
-          size: this.isSurveyGenderAnalysis() ? '60%' : '100%',
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/chart/chart.component.ts:117]: this was swapping out sizes for pie charts. One size for consistency

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**